Documentation/Volumes.md

```diff
@@ -6,24 +6,27 @@ you then provide path to this file together with some metadata.
 NOTE: volumes are only supported for QEMU hipervisor at the moment.
 
 ## Creating a new volume
-You can create a new volume using qemu-img tool:
+You can create a new volume with Capstan:
 
 ```bash
-$ qemu-img create -f qcow2 ./volume.img 1G
+$ capstan volume create volume1
 ```
 
+The command above will create a volume of size 1GB in RAW format by default. Consult `capstan volume create --help`
```

Comment:
I consider qcow2 a better default, don't you? Imagine I make a stupid mistake and create a volume of 100 GB but forget to set the format.